src/app/modules/item/components/add-dependency/add-dependency.component.html

```diff
@@ -0,0 +1,11 @@
+<alg-section icon="pi pi-plus" i18n-label label="Create a new dependency">
```

Comment:
When I disable my network connection and then search for content, the UI appears to be loading infinitely.
And the component does not seem to recover from the error because when I re-enable my network and search for something else, no xhr request is sent.
[link](https://dev.algorea.org/branch/feature/item-dependency-allow-adding/en/#/activities/by-id/4728890134807328169;path=5846722634000078615;attempId=0/details/dependencies)
![image](https://user-images.githubusercontent.com/16257159/184130562-b9b41aa3-9112-497e-a84d-99596907e64f.png)

Reply:
I guess it should be separated issue, as all logic in `AddContentComponent`, used in many places, and seems like it requires to add one more state 'error'

https://github.com/France-ioi/AlgoreaFrontend/blob/853750469f8906b5793f2ee30cbc5d154d0d29b9/src/app/modules/shared-components/components/add-content/add-content.component.ts#L30

Reply:
I've created a https://github.com/France-ioi/AlgoreaFrontend/issues/1129 for it.